### PR TITLE
Add invoice detail fields to Invoice Item

### LINKED ISSUE #1870: Update Invoice Line Item
The Invoice Line item should display the following info:
- Plan that is being paid for
- Duration of the plan
- Total amount charged to the user (instead of showing the plan price).

### PR DESCRIPTION
**Pull request by FSM1**
closes #1870

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

**Comment by asnaith** [comment]
I noticed some minor alignment issues but other than that looks great. 

Screen Shot 2022-01-28 at 8 02 08 PM


PS I found another bug with the pay now link but I've created a separate issue as noticed it was pre-existing #1891

File: packages/files-ui/src/Components/Elements/InvoiceLines.tsx [hunk 1]
```diff
@@ -26,28 +27,30 @@ const useStyles = makeStyles(
...
-              flex: "1 1 0"
-            }
+            marginLeft: constants.generalUnit,
+            marginRight: constants.generalUnit
           }
         }
       },
+      unpaidInvoice: {
+        border: "1px solid #D4380D",
+        color: "#D4380D"
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/InvoiceLines.tsx]: We should have these as `constant` (or `palette` I forgot) so that it can be reused.

**Review by Tbaut — COMMENTED** [review]
Stumbled upon what Andrew mentioned, other than that it looks good!

**Comment by FSM1** [comment]
> I noticed some minor alignment issues but other than that looks great.

Resolved 🎉

**Comment by tanmoyAtb** [comment]
I noticed that the `view pdf` button isn't firing. Sth to do with the button variant changes.

**Comment by FSM1** [comment]
> I noticed that the `view pdf` button isn't firing. Sth to do with the button variant changes.

Were you testing on local or Render? If on local, you need to rebuild the common components.

**Comment by tanmoyAtb** [comment]
> > I noticed that the `view pdf` button isn't firing. Sth to do with the button variant changes.
> 
> Were you testing on local or Render? If on local, you need to rebuild the common components.

Ahh yes. Forgot to build on the PR. 👍

**Review by tanmoyAtb — APPROVED** [review]

**Review by asnaith — APPROVED** [review]

**Pull request merged**